.github/workflows/publish-npm.yml

```
@@ -0,0 +1,36 @@
+name: Publish to npm
+
+on:
+  workflow_dispatch:
```

maybe we can go with:
```suggestion
on:
  release:
    types: [published]
```
so it publishes to npm when a release is published